blackbelt/commands/sprint.py

```diff
@@ -0,0 +1,157 @@
+import click, re, datetime
+from blackbelt.config import config
+
+from blackbelt.handle_trello import get_api, verify, next_week, schedule_list
+
+@click.group(help='Team trello actions for sprints')
+def cli():
+    pass
+
+@cli.command(name='bp')
+def blueprint_plan():
```

Comment:
I am kind of unhappy with the whole logic being in the command directly...it should be only a thin wrapper, akin to controllers. See the other commands.